http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/Boot.scala

```diff
@@ -302,14 +286,18 @@ object Boot extends IOApp {
     )
   }
 
-  private def getSSLContext(implicit actorSystem: ActorSystem) = {
-    val akkaSSLConfig = AkkaSSLConfig()
-    val config = akkaSSLConfig.config
-    val logger = new AkkaLoggerFactory(actorSystem)
-    new ConfigSSLContextBuilder(logger,
-                                config,
-                                akkaSSLConfig.buildKeyManagerFactory(config),
-                                akkaSSLConfig.buildTrustManagerFactory(config)).build()
+  private def getSSLContext()(implicit as: ActorSystem): SSLContext = {
+    val akkaOverrides = as.settings.config.getConfig("akka.ssl-config")
```

Comment:
This is a result of the akka upgrade? IIRC `AkkaSSLConfig()` used to do the config merging automatically. OK with me if it works.

Reply:
yea..`AkkaSSLConfig` is deprecated. The current code is basically copied from akka-http's code without using `AkkaSSLConfig` directly